profiles/profiles.desc

```diff
@@ -249,13 +249,13 @@ x86		default/linux/x86/17.0/uclibc/hardened			exp
 # @MAINTAINER: prefix@gentoo.org
 
 # Linux Profiles
-amd64-linux		prefix/linux/amd64				exp
+amd64-linux		prefix/linux/amd64				dev
```

Comment:
Please remove this modification, because this is `rpath` a profile without glibc, which is not our target.